api/src/certification/evaluation/domain/models/factories/CandidateFactory.js

```diff
@@ -0,0 +1,29 @@
+import { SUBSCRIPTION_TYPES } from '../../../../shared/domain/constants.js';
+import { ComplementaryCertificationKeys } from '../../../../shared/domain/models/ComplementaryCertificationKeys.js';
+import { Frameworks } from '../../../../shared/domain/models/Frameworks.js';
+import { Candidate } from '../Candidate.js';
+
+export class CandidateFactory {
+  static fromCandidateAndSubscription({
```

Comment:
```suggestion
  static fromSubscription({
```

C'est sous-entendu que ce soit "fromCandidate" dans une factory pour l'entité "Candidate" non ?